android/build.gradle

```diff
@@ -10,8 +10,12 @@ android {
     }
 }
 
+def safeExtGet (prop, fallback) {
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+}
+
 dependencies {
     implementation 'com.facebook.react:react-native:[0.40,)'
     implementation 'com.urbanairship.android:urbanairship-fcm:9.5.6'
```

Comment:
we should exclude com.google.firebase:firebase-messaging from this module, and add it with a property like firebaseCoreVersion

Reply:
```
implementation 'com.urbanairship.android:urbanairship-fcm:9.5.6' {
  exclude group: 'com.google.firebase', module: 'firebase-messaging'
}
implementation "com.google.firebase:firebase-messaging':${safeExtGet('firebaseMessaging', '17.3.4')}"
```

Reply:
Added to the PR but I changed it to `firebaseMessagingVersion` for consistency